Website/src/rendering/src/components/Checkout/PromoInput.tsx

```diff
@@ -29,16 +31,22 @@ const PromoInput = (): JSX.Element => {
   };
 
   const addedPromotions = promotions?.length ? (
-    <ul>
+    <ul className="promotion-list">
       {promotions.map((promotion) => {
         return (
-          <li key={promotion.ID}>
-            <button disabled={loading} onClick={() => handleRemovePromotion(promotion.Code)}>
-              Remove Promotion
+          <li key={promotion.ID} className="promotion">
+            <div>
+              <span className="promotion-code">{promotion.Code}</span>
+              <p className="promotion-desc">{promotion.Description}</p>
+            </div>
+            <span className="promotion-amount">-${promotion.Amount}</span>
```

Comment:
Would it be possible for the amount and remove button to be top aligned with the promotion code?